RPIPs/RPIP-42.md

```diff
@@ -72,6 +72,11 @@ This portion of the RPIP SHALL be considered Living. It may be updated by a DAO
 - When showing legacy node status, there is not a trivial way to get the node index for a given address. That said, the other direction is trivial using `RocketNodeManager.getNodeAt`, so the work can get moved off-chain. Eg, one can iterate across all possible node indices and then pass in the node index that matches the node's address; the smart contract can confirm the match to demonstrate legacy node status.
 
 ## Rationale
+The bond curve is an attempt to get close to maximizing capital efficiency while maintaining safety. It balances capital efficiency with slashing, leakage, and MEV theft risks as described in [Security considerations](#security-considerations). 
+
+"Bond reduction" for credit is explicitly supported so that NOs that deposited under higher requirements are not permanently disadvantaged. At the same time, there is no equivalent support for "bond enlargement". This is because those validators are already running and it would need a heavy-handed approach, such as force exiting, to enforce bond enlargement. Since the bond curve is based on total bonded ETH, adding more validators would require the NO to join the active bond curve in such a case. 
```

Comment:
> adding more validators would require the NO to join the active bond curve in such a case.
Wording is a bit awkward. Do you mean if we decide to increase bond size then NO's now have to meet the additional requirement? Could delete that sentence altogether. Or rephrase:
> Since the bond curve is based on total bonded ETH, NO's would be required to add additional ETH bond to maintain their existing validators or add more validators

Reply:
No, they aren't forced to change anything unless they add more validators. They can maintain as is. Maybe:
> Since the bond curve is based on total bonded ETH when depositing, NOs adding new validators would need to add enough ETH bond to match the "enlarged bond" curve.